src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig

```diff
@@ -1234,7 +1234,10 @@ tr.log-status-silenced {
 }
 .sql-explain {
     overflow-x: auto;
-    max-width: 920px;
+    max-width: 888px;
+}
+.width-full .sql-explain {
+    max-width: min-content;
```

Comment:
Here we could even use `max-width: unset` to not restrict the max width, what do you think?

Reply:
Thank you Javier for taking the time to review my pull request. I'm OK with the suggested change.
Let me know if I should make that change. Not sure about the workflow here since it's my first PR.

Reply:
Sorry I didn't reply earlier. Don't worry about this. Now that this is merged, people can test it for real and if there's any issue we'll get reports. So, everything is fine. Thanks!